tests/test_config_tree.py

```diff
@@ -42,27 +42,27 @@
 
 def test_rose_config_exists_no_dir(tmp_path):
     assert rose_config_exists(None, SimpleNamespace(
-        opt_conf_keys=None, defines=[], define_suites=[])
+        opt_conf_keys=None, defines=[], define_workflows=[])
```

Comment:
If you are going to change all of these I'd prefer `workflow_defines` - the original reflected `rose1` syntax, but I dislike it.